### DISCUSSION TITLE
I hope you can also add text to speech and voice cloning features as well

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [GrandaddyShmax/audiocraft_plus#36] [Ideas]
This project looking amazingly promising 

i plan to make a tutorial for this repo hopefully on my channel secourses

https://www.youtube.com/secourses

**Comment by GrandaddyShmax** [comment]
As of right now I would rather focus on audiocraft, but maybe later down the line ill try to add such features and other models outside of audiocraft.

Sorry for taking long time, I honestly forgot to check Discussions tab 😅
